### ISSUE TITLE
Handle TIMESTAMPTZ in Lua SQL queries

### ISSUE THREAD

**Issue by novabyte** [heroiclabs/nakama#219]
The server does not do the required conversion of the SQL TIMESTAMPTZ data type to a format which can be recognised by Lua. This can be demonstrated with:

```lua
local query = [[
SELECT create_time FROM users LIMIT 1
]]
local parameters = {}
local rows = nk.sql_query(query, parameters)
for _, row in ipairs(rows) do
  for k, v in pairs(row) do
    print(k, v)
  end
end
```

As a workaround it's possible to instruct the database to cast the TIMESTAMPTZ to VARCHAR within the query and access the row key with `row["create_time::VARCHAR"]`.

```sql
SELECT create_time::VARCHAR FROM users LIMIT 1
```

**Comment by zyro** [comment]
Fixed in 1c61023c8fc27c8c9e2a8fd306b9d1d57ad45744 and will be part of the next release. Database time types are now converted to UTC seconds before being exposed to the Lua runtime.

This was the last database return value type not correctly handled. In the general case unknown types will now convert to Lua `nil` values - open a new issue for any further types that need special handling.

**Comment by Kobe-Wang** [comment]
Thank you @zyro
